### ISSUE TITLE
[UI idea] Improvements for public adventures

### ISSUE THREAD

**Issue by Felienne** [hedyorg/hedy#4909]
Now that the public adventures are live and I had a chance to play with them, I have a few requests for improvement. We can of course split this into separate issues where needed!

### Update text above adventures
We recently added info text above the adventures, let's also explain public adventures there, with a link to the (yet to be written) teacher manual part about it.
<img width="1120" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/9d2257fe-5d45-40f6-a645-04dceb412b61">

### Change teacher into author?
I am not really sure what the "teacher" column indicates, but I think it means author? An interesting question is if we want to keep the original author there or the author that cloned the adventure? I would lean towards (also?) showing the original author just so it is more clear what is what.

<img width="1131" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/ebc98fd2-b141-4eb4-bfb2-04497bd858ca">

## Show this is a dropdown
I first did not understand this was a dropdown (it misses the arrow down icon). 
<img width="282" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/88669126-0248-4253-9f61-3d11a3f74048">

Of course that has to do with the lack of a general guideline for design of elements, but maybe this is a good moment to centralize the dropdown elements? In other places it does look like a dropdown, f.e. in customization:
<img width="247" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/c7d45f2f-7de9-47ef-9c94-9fcd6c88e208">

## Make tags multiselect
This I think maybe should not be a dropdown, because people might want to search for multiple tags (age and theme, for example) 
<img width="153" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/085db602-13f6-427b-b1fe-17296036855d">
 
## Real preview for preview
Once @jtwaleson is ready with #4822, we can link to a preview in the Hedy UI rather than the edit UI, that would be a lot nicer I think!

## Show a bit more info
A tile now only shows the title and some meta data, but there is some room in the tiles. Could we add maybe the first text line of the adventure, to give a better sense of what it is about?
<img width="374" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/6fdcf89b-3b60-4a6d-8b8d-a0a449f25de8">
